src/sidebar/services/toast-messenger.js

```diff
@@ -10,6 +10,14 @@ const MESSAGE_DISPLAY_TIME = 5000;
 // Delay before removing the message entirely (allows animations to complete)
 const MESSAGE_DISMISS_DELAY = 500;
 
+/**
+ * @typedef ToastMessage
```

Comment:
Can you move this type to the store module? This helps to preserve the overall dependency structure where modules in `services` depend on the `store` but not vice-versa.